src/frequenz/channels/file_watcher.py

```diff
@@ -159,6 +160,10 @@ def __init__(
             watch_filter=self._filter_events,
             force_polling=force_polling,
             poll_delay_ms=int(polling_interval.total_seconds() * 1_000),
+            # Two arguments below are ok for non-Windows systems.
+            # For more details, refer awatch documentation.
+            rust_timeout=sys.maxsize - 1,
+            yield_on_timeout=True,
```

Comment:
https://github.com/samuelcolvin/watchfiles/issues/110

Reply:
But I think that if we change `rust_timeout=sys.maxsize - 1,` back to None.
Then it should work on Windows and won't print debug log. 

This is because the timeout mechanism in awatch looks like this:
```
            if raw_changes == 'timeout':
                if yield_on_timeout:
                    yield set()
                else:
                    logger.debug('rust notify timeout, continuing')
```
So instead of printing debug log, we can yield set() and ignore it. 
Only debug logs won't print if it awatch was killed or not.